server/portal/apps/auth/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.2.5 on 2019-09-12 07:04
+# Generated by Django 2.2.28 on 2022-06-15 00:17
```

Comment:
This brings up a good point. I'm assuming a fresh postgres DB for v3 portals, with any necessary data migrated over. Does this introduce more headaches? I'm thinking it will work out, since much of the data in the postgres DB is tapisv2 related, and we can bring over things like profiles fairly easily.